src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -271,6 +271,7 @@ class BaseOptionsSelector extends Component {
                 isDisabled={this.props.isDisabled}
                 shouldHaveOptionSeparator={this.props.shouldHaveOptionSeparator}
                 onLayout={this.props.onLayout}
+                contentContainerStyles={shouldShowFooter ? [] : [this.props.safeAreaPaddingBottomStyle]}
```

Comment:
is `safeAreaPaddingBottomStyle` a new prop for BaseOptionsSelector? if it is, I think we need to add it to `propTypes` and `defaultProps`

Reply:
@situchan please handle it!

Reply:
done

Reply:
Would it be better to require this param? Is there any instance where we do not want `safeAreaPaddingBottomStyle` passed as a prop to the `BaseOptionsSelector`?

Reply:
```suggestion
                contentContainerStyles={shouldShowFooter ? undefined : [this.props.safeAreaPaddingBottomStyle]}
```

I think it is better `undefined` than `[]`, that way you use the default value for the prop.

Reply:
I think it's better to handle this in low level component. Because it's clear that bottom padding should not be added when footer exists.
`New group` is an example of conditional `safeAreaPaddingBottomStyle`. When any of contacts selected, `Create group` button shows. When no one selected, button is hidden.
Another example is `Split bill` page with conditional render of `Next` button

Reply:
If we wanna pass dynamic `safeAreaPaddingBottomStyle` to `BaseOptionsSelector`, we need this same calculation in parent component:
https://github.com/Expensify/App/blob/a92e18620d8f045464d07805c53a7e70aaa12f63/src/components/OptionsSelector/BaseOptionsSelector.js#L235-L236